components/p3a/brave_p3a_uploader.cc

```diff
@@ -103,31 +50,40 @@ void BraveP3AUploader::UploadLog(const std::string& compressed_log_data,
   resource_request->credentials_mode = network::mojom::CredentialsMode::kOmit;
   resource_request->method = "POST";
 
+  if (resource_request->url.is_empty()) {
```

Comment:
this should not happen in official builds, right? You should CHECK here if it is empty

Reply:
you also need to check to see if it's valid and I would also check `SchemeIsHTTPOrHTTPS`